src/main/java/edu/wpi/first/pathui/FxUtils.java

```diff
@@ -22,4 +25,37 @@ public static MenuItem menuItem(String text, EventHandler<ActionEvent> eventHand
     return menuItem;
   }
 
+  /**
+   * Get the color according to sub-child order.
+   * @param i The index of the child relative to the parent
+   * @return The color to draw the thing as
+   */
+  public static Color getColorForSubChild(int i) {
+    Color[] colors = new Color[]{Color.ORANGE, Color.VIOLET, Color.DARKRED};
```

Comment:
Can you make this a static final member so we don't need to create the array every time?